src/domain/orders/details/returns/receive-menu.tsx

```diff
@@ -36,20 +37,22 @@ const ReceiveMenu: React.FC<ReceiveMenuProps> = ({
   const [quantities, setQuantities] = useState({})
```

Comment:
since `[quantities, setQuantities]` is not used for anything anymore it should be removed